client/src/components/Ratings/RatingBreakdown/RatingList.jsx

```diff
@@ -1,41 +1,78 @@
-import React from 'react';
+/* eslint-disable react/prop-types */
+import React, { useEffect, useState } from 'react';
 import styled from 'styled-components';
-import { useMeta } from '../../../contexts/ReviewMeta';
+import PropTypes from 'prop-types';
 
+import { useMeta } from '../../../contexts/ReviewMeta';
 import RatingBar from './RatingBar';
 
-export default function RatingList() {
+export default function RatingList({ toggleFilter, filterStatus, clearFilter }) {
   const starList = [];
-  // let totalCT = 0;
-
+  const [activeTags, setActiveTags] = useState({
```

Comment:
I always forget you can still pass multiple things in here. Definitely cleaner than adding two pieces of state.